avm/res/machine-learning-services/workspace/version.json

```diff
@@ -1,7 +1,7 @@
 {
     "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
-    "version": "0.2",
+    "version": "0.3",
```

Comment:
I'd suggest to wait for #2590 to avoid conflicts (e.g., on the version update)

Reply:
Will put it in draft for now